doc/tutorials/launch_ubuntu.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Copyright (C) 2019 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+
+function launch_win()
+{
+vm_name=ubuntu_vm$1
+
+logger_setting="--logger_setting console,level=5;kmsg,level=6;disk,level=5"
+
+#check if the vm is running or not
+vm_ps=$(pgrep -a -f acrn-dm)
+result=$(echo $vm_ps | grep "${vm_name}")
+if [[ "$result" != "" ]]; then
+  echo "$vm_name is running, can't create twice!"
+  exit
+fi
+
+#for memsize setting
+mem_size=1024M
+
+acrn-dm -A -m $mem_size -s 0:0,hostbridge \
+  -s 3,virtio-blk,/home/guestl1/acrn-dm-bins/ubuntuUOS.img \
+  -s 4,virtio-net,tap0 \
+  -s 5,virtio-console,@stdio:stdio_port \
+  -k /home/guestl1/acrn-dm-bins/bzImage_uos \
+  -B "earlyprintk=serial,ttyS0,115200n8 consoleblank=0 root=/dev/vda1 rw rootwait maxcpus=1 nohpet console=tty0 console=hvc0 console=ttyS0 no_timer_check ignore_loglevel log_buf_len=16M tsc=reliable" \
+  $logger_setting \
+  $vm_name
+}
+
+# offline SOS CPUs except BSP before launch UOS
```

Comment:
uses of "SOS" should be "Service VM" and "UOS" should be "User VM" throughout the document (unless it's part of a filename or command that is still using the old name and can't be changed.